addons/binding/org.openhab.binding.solarlog/src/main/java/org/openhab/binding/solarlog/internal/HttpUtils.java

```diff
@@ -0,0 +1,81 @@
+
+package org.openhab.binding.solarlog.internal;
+
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.client.util.StringContentProvider;
+import org.eclipse.jetty.http.HttpMethod;
+import org.eclipse.jetty.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+/**
+ * Collection of methods to help retrieve HTTP data from a SolarLog Server
+ * Based on SqueezeBox HttpUtils, adapted for SolarLog.
+ *
+ * @author Dan Cunningham
+ * @author Svilen Valkanov - replaced Apache HttpClient with Jetty
+ * @author Johann Richard - Adapted for SolarLog Binding
+ */
+public class HttpUtils {
+    private static Logger logger = LoggerFactory.getLogger(HttpUtils.class);
```

Comment:
Logger should NOT be static according to the guidelines. [[source](http://docs.openhab.org/developers/development/guidelines.html#e-logging)]